tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -90,168 +69,148 @@ This alternative approach requires that you have npm installed on your machine.
     }
     ```
 
-3. Install AppRouter dependencies using the following commands
-
+3. Within `<destLocation>/approuter` create a new file called `xs-app.json` with the following content:
+    ```json
+    {
+      "welcomeFile": "index.html",
+      "routes": [{
+        "source": "/",
+        "target": "/",
+        "destination": "app-destination"
+        }]
+    }
     ```
-    npm config set @sap:registry https://npm.sap.com
-    npm install
+
+4. Now head to `<destLocation>/firstapp` and replace the `random-route: true` in `manifest.yml` with the following content:
+
+    ```yaml
+    routes:
+      - route: firstapp-<SUBACCOUNT>.cfapps.<REGION_IDENTIFIER>.hana.ondemand.com
```

Comment:
Thinking about it again, I don't think that it is necessary to adapt the route of the backend. The tenant identification is only necessary for the approuter route. As long as the correct URL is entered in destinations below, all should be fine.

Reply:
I believe you are correct. However, I am not sure how random-route behaves when redeploying, especially when you do that by deleting the app in the cockpit and pushing it again. The point here is more to have a stable, unique URL. I'll test if it works with a random route even after redeploying.